src/api/export.ts

```diff
@@ -186,7 +186,7 @@ async function _export({
 					const cleaned = clean_html(body);
 
 					const base_match = /<base ([\s\S]+?)>/m.exec(cleaned);
-					const base_href = base_match && get_href(base_match[1]);
+					const base_href = base_match ? get_href(base_match[1]) : url.pathname;
```

Comment:
Doesn't `pathname` include the the name of the html file? it wouldn't really be the `base_href` in that case

Reply:
good point @benmccann 
I wrapped it with a call to path.dirname, which should do as we need.

Reply:
Actually, it should include the name of the html file. I'm pretty sure that using dirname will not work, so there should be a test for it.

And to make it even better: it should include the search params to make this work correctly: `href="#test"`. (Except that that link probably won't output an extra page)

Reply:
For anchor tags (i.e. `#test`) you would want the name of the html file, but it'd also be repetitive to visit those pages again, so the best approach would probably be discarding those. For normal links to other files I think you'd still want just the directory name, right?

In either case, I agree that a couple more tests would be nice to make sure it's clear those cases are working